themes/socialbase/templates/comment/links--comment.html.twig

```diff
@@ -44,7 +44,7 @@
 
   {% if links['comment-edit'] or links['comment-delete'] or links.report %}
     <div class="comment__actions pull-right">
-      <button type="button" data-toggle="dropdown" aria-expanded="false" aria-haspopup="true" class="btn btn-icon-toggle dropdown-toggle">
+      <button type="button" data-toggle="dropdown" aria-expanded="false" aria-haspopup="true" class="btn btn-icon-toggle dropdown-toggle" name="{{ 'More options'|t }}">
```

Comment:
The `name` attribute is actually used in form submissions and does nothing for accessibility. We should instead use the `aria-label`

See https://developers.google.com/web/tools/lighthouse/audits/button-name